cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  Copyright © 2015 Cask Data, Inc.
+    
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+    
+  http://www.apache.org/licenses/LICENSE-2.0
+    
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+  -->
+
+<configuration>
+
+  <!--
+    Disabling some chatty loggers.
+  -->
+  <logger name="org.apache.commons.beanutils" level="ERROR"/>
+  <logger name="org.apache.zookeeper.server" level="ERROR"/>
+  <logger name="org.apache.zookeeper" level="ERROR"/>
+  <logger name="com.ning" level="WARN"/>
+  <logger name="org.apache.hadoop" level="WARN"/>
+  <logger name="org.apache.hive" level="WARN"/>
+  <logger name="org.quartz.core" level="WARN"/>
+  <logger name="co.cask.cdap" level="INFO"/>
+  <logger name="Explore.stdout" level="INFO"/>
```

Comment:
Can you add a comment explaining what `Explore.stdout` and `Explore.stderr` mean?